backend/inspirehep/records/marshmallow/literature/bibtex.py

```diff
@@ -141,14 +143,14 @@ def get_doi(self, data):
         return get_value(data, "dois.value[0]")
 
     def get_month(self, data):
-        doc_type = data.get("doc_type")
-        date = BibTexCommonSchema.get_date(data, doc_type)
+        doc_type = get_value(data, "document_type[0]")
+        date = self.get_date(data.copy(), doc_type)
         if date:
             return date.month
 
     def get_year(self, data):
-        doc_type = data.get("doc_type")
-        date = BibTexCommonSchema.get_date(data, doc_type)
+        doc_type = get_value(data, "document_type[0]")
```

Comment:
this is wrong, the first document type is not necessarily the most significant one. It's better to use `doc_type`, as that's what will be used by BibTeX.

Reply:
you didn't address this. Please use the bibtex `doc_type` instead (this affects also the map in `get_date` which has to use those).